cmd/kepler/main.go

```diff
@@ -105,3 +112,33 @@ func waitForInterrupt(ctx context.Context, signals ...os.Signal) (func() error,
 			cancel()
 		}
 }
+
+func parseArgsAndConfig() (*config.Config, error) {
+	app := kingpin.New("power-monitor", "Power consumption monitoring exporter for Prometheus.")
+
+	configFile := app.Flag("config.file", "Path to YAML configuration file").String()
+	updateConfig := config.RegisterFlags(app)
```

Comment:
suggestion: change variable name to `updateConfigFn` to indicate this is a function

Reply:
updateConfig is a `verbNoun` and should mean its a func

Reply:
Marking it a resolved.